utils/sane-backends/Makefile

```diff
@@ -174,13 +181,24 @@ define Build/Configure
 	$(Build/Configure/Default)
 endef
 
+define Build/Install
+	$(call Build/Install/Default)
+	mkdir -p $(PKG_INSTALL_DIR)/usr/share/sane
+	$(foreach file, $(filter-out %/unsupported.desc ,$(wildcard $(PKG_BUILD_DIR)/doc/descriptions/*.desc)),
+		sed -rn -e '/^:usbid[[:blank:]]+"?0x(....)"?[[:blank:]]+"?0x(....)"?.*/{s//\1 \2/;p}' $(file) | \
+			sort -u > $(PKG_INSTALL_DIR)/usr/share/sane/$(basename $(notdir $(file))).usbid; \
+		awk '{ print $$$$2 > "$(PKG_INSTALL_DIR)/usr/share/sane/"$$$$1"-$(basename $(notdir $(file))).usbid" '} \
```

Comment:
This throws this warnings under shellcheck, https://github.com/koalaman/shellcheck/wiki/SC1083

Reply:
Ah I see. I think it should be }', not '}

Reply:
Definitely a typo. Thanks.

It was working because the close braces was appended to the single quote string by shell.

I just force-pushed a fix.